lib/repositories/posts/posts_repository_impl.dart

```diff
@@ -154,6 +155,11 @@ class PostsRepositoryImpl extends PostsRepository {
       await _localSettingsSource.save(
           SettingKeys.TX_AMOUNT, currentTxAmount + updatedPosts.length);
     }
+
+    // logs to sentry
+    if (status.value == PostStatusValue.ERRORED) {
+      Logger.log(status.data);
+    }
```

Comment:
I think it's probably better to move this inside the `chainSoure.sendTx` method so that we can log other transactions (like the account-related ones) as well.